frontends/open-discussions/src/util/withTracker.js

```diff
@@ -7,11 +7,12 @@ import ReactGA from "react-ga"
 
 const withTracker = (WrappedComponent: Class<React.Component<*, *>>) => {
   const debug = SETTINGS.reactGaDebug === "true"
-  // $FlowFixMe: process.browser comes from a polyfill provided by webpack
-  const test = !process.browser && process.env.NODE_ENV === "development"
 
   if (SETTINGS.gaTrackingID) {
-    ReactGA.initialize(SETTINGS.gaTrackingID, { debug: debug, testMode: test })
```

Comment:
This is the other application code change. Webpack 5 does not provide process.browser.